tests/test_note.py

```diff
@@ -556,3 +554,49 @@ def test_note_to_from_nonimpl(meth):
 def test_abc_doctave_calc(note, oct, expected):
     fn = partial(_octave_from_abc_parts, base=0)
     assert fn(note, oct) == expected
+
+
+@pytest.mark.parametrize(
+    "scientific,helmholtz",
+    [
+        ("C0", "C,,"),
+        ("C1", "C,"),
+        ("C2", "C"),
+        ("C3", "c"),
+        ("C4", "c'"),
+        ("C5", "c''"),
+        ("B3", "b"),
+        ("B#3", "b#"),
+        ("Cb2", "Cb"),
+    ],
+)
+class TestHelholtz:
+    def test_helmholtz_from_pitch(self, scientific, helmholtz):
+        pitch = Pitch.from_name(scientific)
+        assert pitch.helmholtz == helmholtz
+
+    def test_pitch_from_helmholtz(self, scientific, helmholtz):
+        pitch = Pitch.from_helmholtz(helmholtz)
+        assert pitch.name == scientific
+
+    def test_helmholtz_name_and_octave(self, scientific, helmholtz):
```

Comment:
Not sure if this is what you meant by checking the name...

Reply:
Yes that is what I was thinking, thanks. Sorry it wasn't super clear. Could check `.name` is same too because why not.